.github/actions/verify-pooler-ready/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Verifies that a CNPG cluster has a certain amount of ready instances
+description: Verifies that a CNPG cluster has a certain amount of ready instances
+inputs:
+  pooler-name:
+    description: The name of the pooler to verify
+    required: true
+    default: database-cluster
+  ready-instances:
+    description: The amount of ready instances to wait for
+    required: true
+    default: "3"
+
+runs:
+  using: composite
+  steps:
+    - name: Wait for the pooler to become ready
+      shell: bash
+      run: |
+        ITER=0
+        while true; do
+          if [[ $ITER -ge 300 ]]; then
+            echo "Pooler not ready"
+            exit 1
+          fi
+          READY_INSTANCES=$(kubectl get cluster ${INPUT_POOLER_NAME} -o jsonpath='{.status.readyInstances}')
```

Comment:
```suggestion
          READY_INSTANCES=$(kubectl get pooler ${INPUT_POOLER_NAME} -o jsonpath='{.status.readyInstances}')
```

Reply:
I spotted this as well and fixed it. The check is directly against the pooler deployment now.